android/ReactAndroid/build.gradle

```diff
@@ -256,27 +256,40 @@ def getNdkBuildName() {
 }
 
 def findNdkBuildFullPath() {
-    // we allow to provide full path to ndk-build tool
```

Comment:
unfortunately I don't think any changes to this file are going to persist across SDK upgrades unless we also PR to expo/react-native -- is that something you'd be able to do?

Reply:
`react-native-lab` now contains a commit with this change 😉